### ISSUE TITLE
Organizer does not appear in the attendees

### ISSUE THREAD

**Issue by nraw** [ali-rantakari/icalBuddy#32]
Any way to get the organizer of the event?

**Comment by mlcampbe** [comment]
Did you ever figure out how/if this can be done?

**Comment by nraw** [comment]
@mlcampbe Nope... still hoping for a solution from somewhere. It's often the most critical attendee and yet I can't seem to find a way to obtain it.

**Comment by jjloesch** [comment]
The organizer is an 'attendee' in Apple's Calendar Store https://developer.apple.com/library/archive/documentation/AppleApplications/Conceptual/CalendarStoreProgGuide/Articles/CalendarStoreOverview.html#//apple_ref/doc/uid/TP40004770-SW1
I believe the organizer shall have another meta data associated to the 'attendee' like 'optional' -> 'oranizer' but I'm not sure how to pass it all the way through.
